### PR TITLE
Feat: Style Homepage

### PR DESCRIPTION
**Pull request by renatamottam**
## What's the purpose of this pull request?
Builds and styles the homepage.

> This PR is waiting for these PRs to be merged:
> - https://github.com/vtex-sites/base.store/pull/192
> - https://github.com/vtex-sites/base.store/pull/175

## How it works? 
|Desktop|Mobile|
|-|-|
|screencapture-localhost-8000-2021-12-30-16_02_31|screencapture-localhost-8000-2021-12-30-16_05_04|

### CODE REVIEW

**Review by filipewl — APPROVED** [review]

File: src/components/sections/Hero/Hero.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ const Hero = ({
   return (
     <UIHero>
       <HeroContent aria-labelledby="hero-heading">
-        <div className="banner-info-content">
+        <div className="banner-info-content grid-content">
```
Comment (hellofanny) [inline src/components/sections/Hero/Hero.tsx:31]:
```suggestion
        <div className="banner-info-content / grid-content">
```

File: src/pages/index.tsx [hunk 2]
```diff
@@ -57,32 +59,56 @@ function Page(props: Props) {
...
-        subtitle="Lorem ipsum dolor amet, consectetur adipiscing elit. Lorem ipsum."
-        linkText="See all"
-        link="/"
-        imageSrc="https://storecomponents.vtexassets.com/assets/vtex.file-manager-graphql/images/edce348c-068c-4fb9-91f2-7d235d596e0f___b2822f893b14f87337d08f07f0e520ab.jpg"
-        imageAlt="A person with hands on the pocket, carrying a round straw bag"
-      />
+      <section className="page__section">
+        <Hero
+          title="New Products Available"
+          subtitle="Lorem ipsum dolor amet, consectetur adipiscing elit. Lorem ipsum."
```
Comment (hellofanny) [inline src/pages/index.tsx:65]: Can't we use a fake text instead of lore ipsum? hahahaha

File: src/components/ui/Hero/hero.scss [hunk 3]
```diff
@@ -7,28 +7,37 @@
...
-      max-width: 100%;
       height: 100%;
-      margin-left: auto;
 
       @include media(">=notebook") {
-        max-width: 30rem;
+        padding-left: var(--space-5);
+
+        > div {
+          max-width: 42%;
```
Comment (hellofanny) [inline src/components/ui/Hero/hero.scss:38]: I guess, we might need to tweak these numbers in the future, but for now it's okay :)

**Review by hellofanny — APPROVED** [review]
Looks so beautiful!!! 🤩

**Comment by hellofanny** [comment]
Btw, can you take a look into this issue pls?

image

File: src/pages/index.tsx [hunk 4]
```diff
@@ -57,32 +59,56 @@ function Page(props: Props) {
...
+          linkText="See all"
+          link="/"
+          imageSrc="https://storeframework.vtexassets.com/arquivos/ids/190897/Photo.jpg"
+          imageAlt="A person with hands on the pocket, carrying a round straw bag"
```
Comment (hellofanny) [inline src/pages/index.tsx:69]: We need to update this imageAlt 👀

File: src/components/ui/Hero/hero.scss (hunk 3 above)
Comment (renatamottam) [reply src/components/ui/Hero/hero.scss:38]: 🤡 :mild-panic:

**Pull request merged**